SparkPost/2web2ui
# FE-936 - Re-style summary table global components

[FE-936](https://jira.int.messagesystems.com/browse/FE-936)

### What Changed

- Upgrades to Hibana Matchbox version 15
- Re-styles `<SummaryTable/>` components for Hibana

### How To Test
- The component is used on:
  + `/signals/health-score`
  + `/signals/spam-traps`
  + `/signals/engagement`
- Make sure they continue to behave as they do in production for both the OG app and Hibana
- **Note:** A lot of styles for the tables are derived from the features in which they are used - this includes the use of `<Panel/>` in some spots as well as passed in padding for table cells - this will be addressed during feature cleanup

### To Do
